File: programme/area-of-triangle/area-of-triangle.py
a = float(input('Enter the length of first side: '))
b = float(input('Enter the length of second side: '))
c = float(input('Enter the length of third side: '))

# Calculating the semi-perimeter
s = (a + b + c) / 2

area = (s*(s-a)*(s-b)*(s-c)) ** 0.5

print('The area of the triangle is %0.2f' %area)

File: programme/area-of-triangle/area-of-triangle.cs
// Write a programme to find area of triangle
using System;

public class AreaOfTriangle
{
    static void Main(string[] args)
    {
        Console.Write("Input first side  : ");
        var a = int.Parse(Console.ReadLine() ?? string.Empty);

        Console.Write("Input second side : ");
        var b = int.Parse(Console.ReadLine() ?? string.Empty);

        Console.Write("Input third side  : ");
        var c = int.Parse(Console.ReadLine() ?? string.Empty);

        var s = (a + b + c) / 2;

        Console
            .WriteLine($"\nThe area is : {
                Math.Sqrt(s * (s - a) * (s - b) * (s - c))}");
    }
}
